pkg/globalnet/controllers/base_controllers.go

```diff
@@ -120,7 +120,9 @@ func (c *baseIPAllocationController) reserveAllocatedIPs(federator federate.Fede
 	err := c.pool.Reserve(reservedIPs...)
 
 	if err == nil && len(reservedIPs) > 0 {
+		metricsAllocate(c.pool.GetCider(), len(reservedIPs))
```

Comment:
We should move this to Line 130 as there is a possibility for error during postReserve and the allocated IP can be released in Line127